app/controllers/CategoryController.scala

```diff
@@ -1,80 +1,112 @@
 package controllers
 
-import java.util.Date
+import java.text.{ParseException, SimpleDateFormat}
 
-import com.google.gson.Gson
+import domain.entity.Category
+import domain.models.CategoryModel
 import javax.inject.Inject
+import services.ApiJsonMessage
 import play.api.mvc.{AbstractController, ControllerComponents}
 
 
 class CategoryController @Inject()(cc: ControllerComponents,
-                                   //TODO: add repo here and uncomment returns in methods https://trello.com/c/wWclhRQx/103-add-category-repository
-                                   gson: Gson)
+                                   model: CategoryModel)
   extends AbstractController(cc){
 
+  private val message = new ApiJsonMessage()
+
   def getAll = Action {
-    //Ok(repo.getAll())
-    throw new UnsupportedOperationException()
-    Ok("")
+    try{
+      Ok(model.getAll)
+    }
+    catch {
+      case e: Exception =>
+        InternalServerError(message.create(e.getLocalizedMessage.replace("\"", "'")))
+    }
   }
 
   def getOne(id: String) = Action {
```

Comment:
@Bukarachka "FindBy" is more valuable naming, also it's more widely used. Let's rename it to "findBy"